static/js/containers/App.js

```diff
@@ -282,7 +283,8 @@ const mapStateToProps = state => {
     currentProgramEnrollment: state.currentProgramEnrollment,
     programs:                 state.programs,
     courseEnrollments:        state.courseEnrollments,
-    signupDialog:             state.signupDialog
+    signupDialog:             state.signupDialog,
+    shareDialog:              state.shareDialog
```

Comment:
same here, trailing comma would be useful.

Reply:
fmt doesn't think that there should be a comma there.

Reply:
interesting, you can merge it 👍